data/a-a.ts

```diff
@@ -0,0 +1,32 @@
+// TODO
+const TEST_CFR = {};
+
+export default {
+  name: "A/A test",
+  description: "Use this to test audiences or to diagnose issues in the system.",
+  presets: {
+    // Specific constraints for the ui
+    max_bucket_count: 100,
+
+    // Defaults for specific experiment values
+    branches:  [
+      {
+        slug: "control",
+        is_reference_branch: true,
+        ratio: 1,
+        value: TEST_CFR,
```

Comment:
Maybe for this first test, we make this simpler and include an empty payload for now.